### PR TITLE
/usr/local/Cellar/grep 目录下只有grep

### PR DESCRIPTION
**Pull request by M2shad0w**
image

### CODE REVIEW

**Comment by wufeifei** [comment]
请正确安装ggrep

**Comment by M2shad0w** [comment]
image

[现在最新的rb轮子是默认不带g前缀 homebrew grep](https://github.com/Homebrew/homebrew-dupes/blob/master/grep.rb)

**Comment by M2shad0w** [comment]
brew install grep --program-prefix=g

File: engine/static.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def analyse(self):
             gfind = ''
             for root, dir_names, file_names in os.walk('/usr/local/Cellar/grep'):
                 for filename in file_names:
-                    if 'ggrep' == filename:
```
Comment (wufeifei) [inline engine/static.py]: 感谢反馈，请增加兼容处理，并恢复其它几个文件的改动后再提交PR。
`if 'ggrep' == filename or 'grep' == filename:`

**Pull request closed without merging**